gamemodes/base/gamemode/gravitygun.lua

```diff
@@ -0,0 +1,36 @@
+if SERVER then
+	include( "cfc_err_forwarder/init.lua" )
+end
```

Comment:
Maybe put this at the bottom of the file so errors in our code don't halt this file? Idk if that's actually a concern.

A comment explaining what we're doing here might also be nice